grakn-engine/src/test/java/ai/grakn/engine/SystemKeyspaceTest.java

```diff
@@ -38,33 +40,52 @@
 import java.util.Arrays;
 import java.util.HashSet;
 import java.util.Set;
+import java.util.concurrent.locks.Lock;
 import java.util.function.Function;
 import java.util.stream.Collectors;
 
 import static ai.grakn.engine.SystemKeyspace.SYSTEM_KB_KEYSPACE;
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 
 public class SystemKeyspaceTest {
-    private final Function<String, GraknTx> engineFactoryGraphProvider = (k) -> EngineTestHelper.factory().tx(k, GraknTxType.WRITE);
-    private final Function<String, GraknTx> externalFactoryGraphProvider = (k) -> Grakn.session(EngineTestHelper.uri(), k).open(GraknTxType.WRITE);
+    private final Function<String, GraknTx> engineFactoryKBProvider = (k) -> graknFactory.tx(k, GraknTxType.WRITE);
+    private final Function<String, GraknTx> externalFactoryGraphProvider = (k) -> Grakn.session(sparkContext.uri(), k).open(GraknTxType.WRITE);
 
+    private static final GraknEngineConfig config = GraknEngineConfig.create();
+    private static final GraknEngineStatus status = mock(GraknEngineStatus.class);
+    private static final MetricRegistry metricRegistry = new MetricRegistry();
+    private static final LockProvider lockProvider = mock(LockProvider.class);
+    private static final EngineGraknTxFactory graknFactory = EngineGraknTxFactory.createAndLoadSystemSchema(lockProvider, config.getProperties());
```

Comment:
I think this might need to go in a `setUp` method, so it's called after cassandra is definitely started.